README.md

```diff
@@ -1,2 +1,39 @@
 # signoff-pxb
 Repos for pxb related release sign-off
+
+Each PX-Backup release need to follow this sign-off process and share all artifacts before releasing to production.
+PX team need to share the compatibility metrics as well for release. 
+
+
+## Following are the tests scope for PX-Backup
+
+- Installation tests
+- Torpedo tests
+- Functional tests
+- Market place tests
+- Integration tests
+
+### [Owner PX team] Tests that must be executed in the staging environment and performed with every release of px-backup:
+
+- Installation tests
+- Torpedo tests
+- Functional tests
```

Comment:
This should be signed off by Siva. Please check with him.

Reply:
@ambiknai , can we ask him

Reply:
We maintain the functional tests in our test rail for each release and also automate them as much as possible. We can share the results of those run.

Reply:
@trenukarya-px , Will IBM will have access to our test rail?